src/storage/mod.rs

```diff
@@ -341,7 +349,8 @@ mod test {
     fn test_batched_uploads() {
         let uploads: Vec<_> = (0..=MAX_CONCURRENT_UPLOADS + 1)
             .map(|i| {
-                let (content, alg) = compress("fn main() {}".as_bytes()).unwrap();
+                let alg = DEFAULT_COMPRESSION;
+                let content = compress("fn main() {}".as_bytes(), alg).unwrap();
```

Comment:
Again it seems strange to set a variable to `DEFAULT_COMPRESSION`

Reply:
See the other comment.